include/sway/commands.h

```diff
@@ -39,6 +42,9 @@ enum expected_args {
 	EXPECTED_EQUAL_TO
 };
 
+void input_cmd_apply(struct input_config *input);
+void seat_cmd_apply(struct seat_config *seat);
```

Comment:
Can this be moved in `config.h`, just like `apply_output_config`? (Maybe rename the latter)

Reply:
Yeah we need to do a big name cleanup because there are like 3 different conventions going on in the project right now.